src/main/scala/firrtl/FileUtils.scala

```diff
@@ -118,9 +118,9 @@ object FileUtils {
 
   /** Read a text file and return it as  a Seq of strings
     * Closes the file after read to avoid dangling file handles
-    * @todo deprecate java.io.File
     * @param file a java File to be read
     */
+  @deprecated("Use getLines(file: os.Path)", "FIRRTL 1.4.3")
```

Comment:
`@deprecated("Use os-lib instead, this function will be removed in FIRRTL 1.6", "FIRRTL 1.5")`

Reply:
Fixed in 58c6479